akka-persistence-query/src/main/scala/akka/persistence/query/EventEnvelope.scala

```diff
@@ -12,3 +12,14 @@ final case class EventEnvelope(
   persistenceId: String,
   sequenceNr:    Long,
   event:         Any)
+
+/**
+ * Event wrapper adding meta data for the events in the result stream of
+ * [[akka.persistence.query.scaladsl.EventsByTagQuery2]] query, or similar queries.
+ */
+// TODO: Rename it to EventEnvelope in Akka 2.5
```

Comment:
I think there is one already, as this was copied from the #21615 changes, but I will double check.